src/functions/reportGen.ts

```diff
@@ -16,18 +16,20 @@ import { ActivitySchema } from "@dvsa/cvs-type-definitions/types/v1/activity";
  * @param context - λ Context
  * @param callback - callback function
  */
-const reportGen: Handler = async (event: any, context?: Context, callback?: Callback): Promise<void | PutObjectRequest[]> => {
+const reportGen: Handler = async (event: any, context?: Context, callback?: Callback): Promise<DynamoDBBatchResponse> => {
```

Comment:
Little confused at the types here, so apologies if this is not correct... 

But shouldn't the types here be 
`const reportGen = async (event: SQSEvent): Promise<SQSBatchResponse> => {`

If that is the case, it highlights an issue with the logic inside the catch.